runbotics/runbotics-orchestrator-ui/src/src-landing/views/MarketplaceCartView/MarketplaceCartView.tsx

```
@@ -42,6 +42,29 @@ const MarketplaceCartView: FC = () => {
             ...contactFormValue,
             cartContent: selectedFullCartItems,
         };
+
+        if (contactFormValue.name.trim().length === 0) {
+            enqueueSnackbar(
+                translate('Marketplace.Cart.NoNameAndSurnameError'),
+                {
+                    variant: 'error',
+                    autoHideDuration: 5000,
+                },
+            );
+            return;
+        }
+
+        if (!/^\S+@\S+\.\S+$/.test(contactFormValue.email)) {
+            enqueueSnackbar(
+                translate('Marketplace.Cart.NoEmailError'),
+                {
+                    variant: 'error',
+                    autoHideDuration: 5000,
```

and you can use this const here. also - you can think of moving this to runbotics-common and we could use it in different places going further as for now there is no default auto hide duration and I can see in different places it is subjectively added